src/components/Analyze/AnayzeChoice.tsx

```diff
@@ -1,27 +1,62 @@
+import { AnalyzeContext } from '@/components/Analyze/\bAnalyzeContext'
+import { useContext } from 'react'
+
+const addComma = (price: number) => {
+	let returnString = price?.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',')
+	return returnString
+}
+
 export const AnalyzeChoice = () => {
+	const { totalPrice } = useContext(AnalyzeContext)
 	return (
-		<div className='flex flex-initial tablet:flex-row gap-4 justify-start w-full'>
-			<div className='flex-1 flex flex-col justify-center items-center min-w-[15rem] w-full'>
-				<label className='flex-1' htmlFor='날짜'>
-					날짜
-				</label>
-				<div className='flex flex-col tablet:flex-row flex-1 border-2 p-1 rounded-3xl'>
-					{/* datepicker 사용 예정 */}
-					<input type='date' />
-					<input type='date' />
+		<div className='flex flex-initial flex-col tablet:flex-row gap-1 justify-start w-full tablet:gap-10 px-10'>
+			<div className='relative flex-1 flex flex-col justify-start items-center bg-[#f6f6f7] rounded-[10px] shadow-analyze-box w-full h-full p-[20px]'>
+				<div className='flex flex-col justify-between items-center tablet:flex-row flex-1 gap-2 w-full'>
+					<div className='flex flex-1 flex-col'>
+						<label
+							className='text-[#838383] text-base font-medium tracking-wide'
+							htmlFor='시작일'
+						>
+							시작일
+						</label>
+						<input
+							type='date'
+							className='text-2xl font-medium text-[#554a4a]'
+							value='2025-01-01'
+						/>
+					</div>
+					<div className='flex flex-col flex-1'>
+						<label
+							className='text-[#838383] text-base font-medium tracking-wide'
+							htmlFor='종료일'
+						>
+							종료일
+						</label>
+						<input
+							className='text-2xl font-medium text-[#554a4a]'
+							type='date'
+							value='2025-01-01'
+						/>
+					</div>
 				</div>
 			</div>
-			<div className='flex flex-col flex-1 items-center text-center min-w-[15rem]'>
-				<label className='flex-1' htmlFor='날짜'>
-					분류
-				</label>
-				<div className='flex gap-2 justify-center border-2 p-1 rounded-3xl'>
-					<button className='flex-initial rounded-2xl p-0.5 text-sm'>
-						카테고리
-					</button>
-					<button className='flex-initial rounded-2xl p-0.5 text-sm'>
-						장소
-					</button>
+			<div className='relative flex flex-1 flex-row justify-start items-center bg-[#f6f6f7] rounded-[10px] shadow-analyze-box w-full h-full p-[20px]'>
+				<div className='ml-3 flex flex-1 flex-col'>
+					<label
+						className='text-[#838383] text-base font-medium tracking-wide'
+						htmlFor='총 지출액'
+					>
+						총 지출액
+					</label>
+					<img
+						src='./src/images/totalPrice.svg'
+						className='w-[34px] h-[32px]'
+					></img>
+				</div>
+				<div className='flex justify-between mr-5'>
+					<div className='font-semibold text-2xl'>
+						{addComma(totalPrice ? totalPrice : 0)}원
```

Comment:
0원을 표기하기 위함이라면 null or undefined만을 처리하는 `??` 널 병합 연산자를 추천드립니다.
```suggestion
{addComma(totalPrice ?? 0)}원
```